jack_bot.py

```diff
@@ -137,69 +140,7 @@ def __init__(self, debug=False):
         self.locations = []
         self.ordered_expansions = []
         self.building_positions = []
-        self.close_enemies_to_base = False
-        self.close_enemy_production = False
-        self.floating_buildings_bm = False
-        self.counter_attack_vs_flying = False
```

Comment:
this is missing on the refactoring

Reply:
please review these changes in here https://github.com/Matuiss2/Greedy_bot/pull/148